packages/backend/src/peripherals/database/AggregatedReportRepository.ts

```diff
@@ -21,24 +22,27 @@ export class AggregatedReportRepository extends BaseRepository {
   async getDaily(): Promise<AggregatedReportRecord[]> {
     const knex = await this.knex()
     const rows = await knex('aggregated_reports')
-      .where('is_daily', true)
```

Comment:
Shouldn't the selector start with `where` instead of `andWhere`?
Something like this: `knex.where(x).andWhere(y)`
Instead of: `knex.andWhere(x).andWhere(y)`

If so, we should be able to write a test that fails when we do the incorrect thing?

Reply:
Knex is smart enough to concat those in the proper manner:
```ts
    const sql = knex('aggregated_reports')
      .andWhere('unix_timestamp', '>=', from.toDate())
      .andWhere({ value_type: ValueType.TVL.toString() })
      .andWhereRaw(`EXTRACT(hour FROM unix_timestamp) % 6 = 0`)
      .orderBy('unix_timestamp')
      .toSQL()

    console.dir({ sql })
```

```bash
{
  sql: {
    method: 'select',
    options: {},
    timeout: false,
    cancelOnTimeout: false,
    bindings: [ 2023-07-25T08:06:16.000Z, 'TVL' ],
    __knexQueryUid: 'OJA882bwhjTg_0DR962rZ',
    sql: 'select * from "aggregated_reports" where "unix_timestamp" >= ? and "value_type" = ? and EXTRACT(hour FROM unix_timestamp) % 6 = 0 order by "unix_timestamp" asc'
  }
}
```

Simply fixing it here is enough, good catch

Reply:
https://github.com/l2beat/l2beat/pull/1716/commits/7702e113b3ff2ff54e3abc0241dbd0f50a360e22